Tiltfile

```
@@ -108,6 +115,12 @@ COPY --from=tilt-helper /usr/bin/kubectl /usr/bin/kubectl
     },
 }
 
+def ensure_envsubst():
+    local("make " + envsubst_cmd)
+
+def ensure_clusterctl():
+    local("make " + clusterctl_cmd)
```

Doesn't work atm:

```
Traceback (most recent call last):
  /home/sbuerin/code/src/sigs.k8s.io/cluster-api/Tiltfile:513:12: in <toplevel>
  /home/sbuerin/code/src/sigs.k8s.io/cluster-api/Tiltfile:405:22: in prepare_all
  /home/sbuerin/code/src/sigs.k8s.io/cluster-api/Tiltfile:122:10: in ensure_clusterctl
  <builtin>: in local
Error: command "make ./bin/clusterctl" failed.
error: exit status 2
```

```suggestion
    local("make clusterctl")
```

Fine for me to keep it easy and hard-code it